pipeline/data.py

```diff
@@ -24,7 +32,7 @@ def new_time(t, t_start=None, t_end=None):
     local=LOCAL,
     region=REGION,
     vm_type="m6i.2xlarge",
-    tags={"workflow": "etl-tpch"},
```

Comment:
why remove the tag? it seems nice to have tag on these.

Reply:
The workspace we run this is in called `etl-tpch-prod`, so the tag feels a little redundant. I suppose that's just for us though -- if somebody else runs this, then that wouldn't be the case. I can add these back in a little bit